examples/multiple-choice/index.es

```diff
@@ -0,0 +1,77 @@
+Element `multiple-choice`
```

Comment:
https://github.com/devpunks/snuggsi/blob/master/dist/snuggsi.es#L484 this it @snuggs ?